packages/components/src/components/typography/Heading/Heading.tsx

```diff
@@ -27,36 +28,28 @@ const baseStyles = css`
     line-height: normal;
```

Comment:
line-height and spacing are already defined in `typography`, no need to have them here.